xcode/Subconscious/Shared/Services/Noosphere/Sphere.swift

```diff
@@ -194,16 +194,16 @@ public final class Sphere: SphereProtocol {
         return Self.readFileHeaderNames(file: file)
     }
     
-    /// Read the value of a memo from this Sphere
+    /// Read the value of a memo from this, or another sphere
     /// - Returns: `MemoData`
-    private func read(slug: Slug) throws -> MemoData {
+    public func read(slashlink: Slashlink) throws -> MemoData {
         guard let file = try Noosphere.callWithError(
```

Comment:
Sort of an aside, but I wonder if it is possible in Swift to curry this so that you can invoke the FFI more idiomatically like a straight-up function call (but also benefit from the semantics of `callWithError`). (no action required)